tensorflow/inference/docker/2.5/py3/Dockerfile.sagemaker.cpu

```diff
@@ -0,0 +1,33 @@
+# Expecting base image to be the image built by ./Dockerfile.diy.cpu
+ARG BASE_IMAGE=""
```

Comment:
nit: `DIY_IMAGE` can be used instead of `BASE_IMAGE` to be very clear on the working on SM images

Reply:
This arg is explicitly referenced by our build code. I think we should keep this as `BASE_IMAGE`, since it allows modularity for other staged builds. I added a comment on top which details the expected base image